packages/react-core/src/components/Form/Form.tsx

```diff
@@ -9,24 +9,34 @@ export interface FormProps extends React.HTMLProps<HTMLFormElement> {
   className?: string;
   /** Sets the Form to horizontal. */
   isHorizontal?: boolean;
-  /** Flag to limit the max-width to 500px. */
+  /** Limits the max-width of the form. */
   isWidthLimited?: boolean;
+  /** Sets a custom max-width when using isWidthLimited. */
```

Comment:
Nit: I think `isWidthLimited` should be sentence case.

Reply:
Not sure if I understand - can you add the code suggestion change to the comment?

Reply:
Actually I just removed the refence to `isWidthLimited` part now that `maxWidth` applies `.pf-m-limit-width`. wdyt?